akka-docs-dev/rst/scala/stream-flows-and-basics.rst

```diff
@@ -161,7 +161,7 @@ Fast Publisher, slow Subscriber
 This is the case when back-pressuring the ``Publisher`` is required, because the ``Subscriber`` is not able to cope with
 the rate at which its upstream would like to emit data elements.
 
-Since the ``Publisher`` is not allowed to signal more elements than the pending demand signalled by the ``Subscriber``,
+Since the ``Publisher`` is not allowed to produce more elements than the pending demand signalled by the ``Subscriber``,
```

Comment:
Hm, so `to signal elements` is reactive streams wording (used in the spec). Let's keep it as we're discussing reactive streams things in this paragraph